frontend/src/pages/HistoryPage.vue

```diff
@@ -102,10 +101,10 @@ export default {
         ]),
 
         mapInitialize(map) {
-            this.markerManager = new MarkerService(map);
+            this.markerManager = new MarkerManager.getService(map);
```

Comment:
your method has already returned a new instance, you don't need to use `new` here, and change first letter to the small one